frontend/src/screens/Home.tsx

```diff
@@ -1,33 +1,200 @@
+import { Card, IconButton, Text } from 'react-native-paper'
+import { getPlaylist } from '../services/user.service'
+import { Track, trackItemMapper } from '../util/track'
+import { Audio, AVPlaybackStatusSuccess } from 'expo-av';
+import Slider from '@react-native-community/slider'
+import { themeContext } from '../providers/theme.provider'
+import { millisToHHMMSS } from '../../helpers'
 
 const Home = ({ navigation }): JSX.Element => {
-  // useEffect(() => {
-  //     if (cookies.loginCookie !== '') {
-  //         axios.get(
-  //             "https://api.spotify.com/v1/me/top/tracks?time_range=short_term", {
-  //                 headers: {
-  //                     Accept: "application/json",
-  //                     "Content-Type": "application/json",
-  //                     Authorization: "Bearer " + cookies.loginCookie,
-  //                 },
-  //             }).then((response: any) => {
-  //             // console.log(response)
-  //         })
-  //             .catch((error: { message: any; }) => {
-  //                 console.log("error", error.message);
-  //                 removeCookie('loginCookie');
-  //             });
-  //     }
-  // }, [])
-
-  return (
-    <View>
-      <Text>you're logged in!</Text>
-      {/* <Button onPress={() => logOut()} title={'logOut'}/>
-            <Button onPress={() => navigation.navigate('Landingpage')} title={'go back to landingPage'}/> */}
+  const { theme } = useContext(themeContext)
+  const [selectedTrack, setSelectedTrack] = useState<Track>();
+  const [tracks, setTracks] = useState<Track[]>([])
+  const audioSoundRef = React.useRef(new Audio.Sound());
+  const [soundStatus, setSoundStatus] = useState<AVPlaybackStatusSuccess>()
+  const [isPlaying, setIsPlaying] = useState<boolean>(false)
+
+
+  useEffect(() => {
+    getPlaylist("1LwKg8pkx71G83WgOfvlLZ").then((data) => {
+      setTracks(trackItemMapper(data.items))
+    })
+  }, [])
+
+  const BOTTOM_TAB_HEIGHT = 77.71
+  const PLAYBACK_HEIGHT = 40
+  const CARD_WIDTH = Dimensions.get('window').width
+  const CARD_HEIGHT = Dimensions.get('window').height - BOTTOM_TAB_HEIGHT - PLAYBACK_HEIGHT
+  const styles = StyleSheet.create({
+    cardContentStyle: {
+      flex: 1,
+      alignItems: 'center',
+      justifyContent: 'center'
+    },
+    cardStyle: {
+      width: CARD_WIDTH,
+      height: CARD_HEIGHT,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      // backgroundColor: 'teal',
+      borderRadius: 15
+    },
+    title: {
+      textAlign: 'center',
+    },
+    image: {
+      width: 256,
+      height: 256,
+      alignItems: 'center',
+      borderRadius: 0,
+    },
+    cardTitle: {
+      marginLeft: 5,
+    },
+    subtitle: {
+      marginLeft: 5,
+    },
+    header: {
+      marginTop: 10,
+      marginVertical: 10,
+      marginHorizontal: 10,
+    },
+  })
+
+  const stop = async () => {
```

Comment:
Please make components and or a provider out of this. You are putting business logic in a main page ...